airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -213,24 +218,32 @@ private CancellationScope generateSyncWorkflowRunnable(final ConnectionUpdaterIn
         prepareForNextRunAndContinueAsNew(connectionUpdaterInput);
       }
 
-      // TODO (pedroslopez): The feature flags will actually be used in a future PR
-      Map<String, Boolean> featureFlags = getFeatureFlags(connectionUpdaterInput.getConnectionId());
+      final Map<String, Boolean> featureFlags = getFeatureFlags(connectionUpdaterInput.getConnectionId());
+      workflowInternalState.setFeatureFlags(featureFlags);
```

Comment:
Note: in order to more easily use feature flags I've added them to the internal workflow state

Reply:
this caused conflicts with replaying old workflows, so did not do this.